### ISSUE TITLE
Web page fails HTML5 markup validation

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#9]
**URL**: https://hcdev.openplus.ca/en
**User Agent** Chrome/FF
**language**: both

**Issue Description**: Web page fails HTML5 markup validation. Generated HTML source is used to validate the markup using W3C Validator (https://validator.w3.org/). 

**Steps to reproduce:**
1. In Chrome or Firefox, press F12 key to open the Web Developer Tools. 
2. In the code, right click on <html> element and select “Copy element”. 
3. In W3C validator, click “Validate by direct input” tab. 
4. Type “<!DOCTYPE html>” and paste code copied below. 
5. Click "Check" button and go through the warnings/errors. There shouldn't be any errors.

**Expected results**: All errors should be fixed.

**Comment by qanitah** [comment]
Other pages fail validation as well. For example, Home, Edit profile, etc. Please review all and all errors should be addressed.

**Comment by ndolter** [comment]
Validated that no errors exists.
